com.playeveryware.eos/Documentation~/full_guide/01_starthere.md

```diff
@@ -0,0 +1,126 @@
+See [Epic's documentation on Get Started Overview](https://dev.epicgames.com/docs/epic-games-store/get-started/get-started-overview#onboarding-process) for more information.
+While this agreement is required to perform Epic Game Store operations, including publishing and selling your game, the EOS Plugin for Unity can be configured and utilized before this is agreed to.
+
+# Creating a Client Policy and Client
+
+The landing page for a Product will note your existing Product, Clients, Sandboxes, and Deployments.
+
+<img src="/com.playeveryware.eos/Documentation~/full_guide/epic_developerportal_productlanding.png" alt="The first thing in the Product landing page is your Product, Clients, Sandboxes, and Deployments. Other than Product, these likely haven't been configured yet."/>
+
+In order to get the most out of the EOS Plugin for Unity, you'll need to create several of these.
+Start by creating a Client by navigating to the `Clients` section.
+There you'll see your list of Clients and Client policies, which will be empty.
+
+For more information on Clients and Client Policies, see [Epic's Client Credential and Client Policy Management documentation](https://dev.epicgames.com/docs/dev-portal/client-credentials).
+Clients are essentially types of contexts for your game. For example, all users running your game from the store will be assigned a Client that represents their abilities.
+Each Client is assigned to one Client Policy.
+Client Policies are configurable sets of rules for what permissions a Client can perform. For example, a Client Policy determines the EOS SDK's ability to look up information about a player's friend list.
+
+First, create a Client policy. Press the `Add new policy` button to bring up the dialogue.
+The Client Policy needs a name, as well as select the details of a policy.
+If you select a preset, the permissions for that policy preset will be displayed in the dialogue.
+Custom policies can be defined here, and changed at any time. You don't need to commit to the final details at this time.
+The most permissive default policy is `GameClient /w UnlockAchievements`, which will give the Client Policy all of the permissions necessary to perform most actions.
+
+When the Client Policy is created, next create a client. Press `Add new client` to bring the Add new client dialogue.
+This will require a name and a Client Policy. Use the previously created Client Policy.
+`Trusted Server IP allow list` and `Epic Account Service Redirect URL` are not required to be filled at this time.
+
+# Creating an Application
+
+Navigate to the `Epic Account Services` page. The landing will show you all configured Applications, which is empty at first.
+See [Epic's documentation on Applications in the Developer Portal Introduction](https://dev.epicgames.com/docs/dev-portal/dev-portal-intro#application) for more information on Applications.
```

Comment:
Again, more information links and references should probably happen toward the end instead of the beginning.

Reply:
I'm not certain how to follow this guidance in concert with other guidance provided. Where would you put this? At the end of the section? Or linked when Applications are first mentioned?

Reply:
Fair point. Wait... did you expect me not to contradict myself constantly? 🤪